default.nix

```diff
@@ -1,5 +1,5 @@
 { sources ? import nix/sources.nix
-, channelName ? "nixpkgs-unstable"
+, channelName ? "nixpkgs-22.11"
```

Comment:
When we do the build in https://github.com/replit/goval/blob/main/scripts/nixpkgs_update#L118 it will specify the channel individually. So this won't affect that.